boefjes/boefjes/plugins/kat_ssl_test_ciphers/description.md

```diff
@@ -0,0 +1,64 @@
+# Nmap
```

Comment:
Copied over from nmap, needs own description.

Reply:
Dont know how i missed that, apologies.